designs/virtualization/4.6/Upload-image-prior-to-creating-VM/Upload Image prior to creating vm.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.6
+---
+# Upload Image prior to creating a VM
+
+This PR deals with when a user wants to upload an image prior to creating a virtual machine.
+It needs to be accomplished via the storage/PVC area.
+It should ensure the user understands what the backing PVC does but does not remove the Disk terminology.
+
+The ‘Create PVC’ button will change into a dropdown, so users will be able to select ‘Upload data to Persistent Volume Claim’ from this dropdown. 
+
+![PVC list with drop down button](img/PVC-dropdown.png)
+
+They will get to a new upload screen with an inline info alert that says this PVC will be created using a Data Volume through Container Data importer (CDI). 
+User chooses an image to upload and fills in the PVC details: Name, size, storage class, access mode. Some of the fields can be filled out but editing will be optional as well.
+
+![browse to upload](img/Upload-data-to-pvc-1.png)
```

Comment:
Should access mode have a default option selected since it's a radio button group?

Reply:
AFAIU the access mode is set by the storage class and cannot be changed. @tjelinek@redhat.com is it correct?

Reply:
+1 this should show single user as selected right If the others are disabled?

Reply:
:) sure, thanks